lib/pure/times.nim

```diff
@@ -1344,16 +1658,32 @@ proc format*(dt: DateTime, f: string): string {.tags: [].}=
 
     inc(i)
 
+proc format*(time: Time, f: string): string {.tags: [].} =
+  ## converts a `Time` value to a string representation. It will use the local
+  ## time zone and use the format from ``format(dt: DateTime, f: string)``.
```

Comment:
> use the format from ``format(dt: DateTime, f: string)``.

Huh? I don't understand this.

Reply:
oh. I see what you mean after a while.

This function shouldn't exist, developers can easily convert to a DateTime. Please don't create small functions that add such a small convenience, it just adds complexity and further mental overhead.

Reply:
Disagree, it is extremely common to format a time, every other language allows that, and users will expect to be able to just do that

Reply:
Agreed with andreaferretti. It's a wrapper, yeah. But i've created it because I want to be able to use such functions 'out of the box', without a need to convert anything. Why should i? Cause someone in a random point in time has decided that `DateTime` is somehow more important, than the `Time` is? Nonsense.

Reply:
> Why should i? Cause someone in a random point in time has decided that DateTime is somehow more important, than the Time is?

Because ``Time`` has no timezone information. I'd be more comfortable forcing developers to explicitly specify they want the "local" timezone instead of implicitly choosing it for them. Implicit decisions like this could lead to subtle issues.

Additionally, this conversion couldn't be simpler:

```nim
time.local.format(f)
```